docs/source/howto/ssh.rst

```diff
@@ -206,24 +224,30 @@ Edit the ``~/.ssh/config`` file on the computer on which you installed AiiDA (or
 
 replacing the ``..._TARGET`` and ``..._PROXY`` variables with the host/user names of the respective servers.
 
-This should allow you to directly connect to the *TARGET* server using
+In both cases, shis should allow you to directly connect to the *TARGET* server using
 
 .. code-block:: console
 
    $ ssh SHORTNAME_TARGET
 
 For a *passwordless* connection, you need to follow the instructions :ref:`how-to:ssh:passwordless` *twice*: once for the connection from your computer to the *PROXY* server, and once for the connection from the *PROXY* server to the *TARGET* server.
 
-.. dropdown:: Specifying an SSH key for the proxy
+.. dropdown:: Specifying an SSH key for the proxy when using ``ProxyCommand``
 
    If you need to specify a separate SSH key for the proxy, provide it *after* the ``-W`` directive, e.g.::
 
         ssh -W fidis.epfl.ch:22 -i /home/ubuntu/.ssh/proxy user@proxy.epfl.ch
 
+
 AiiDA configuration
 ^^^^^^^^^^^^^^^^^^^
 
-When :ref:`configuring the computer in AiiDA <how-to:run-codes:computer:configuration>`, AiiDA will automatically parse the required information from your ``~/.ssh/config`` file.
+When :ref:`configuring the computer in AiiDA <how-to:run-codes:computer:configuration>`, AiiDA will automatically parse most of required information from your ``~/.ssh/config`` file.
+
+.. dropdown:: Specifying the proxy_jump
+
+    The ``proxy_jump`` *must** be specified manually when configuring the SSH connection and is not automatically taken from your ``~/.ssh/config``.
```

Comment:
will it figure out to use the correct key for the proxy?

Reply:
I added a comment: with the current design the same key is used (if any) as for the main/target host.
Theoretically you could specify a complete set of configurations for each proxy jump (`ssh_config` allows that).
But that would make the interface rather complicated, so I went with 80/20 rule and simply copy the complete config from the main host.